cmake/bintools/gnu/target.cmake

```diff
@@ -12,6 +12,23 @@ find_program(CMAKE_NM      ${CROSS_COMPILE}nm      PATHS ${TOOLCHAIN_HOME} NO_DE
 find_program(CMAKE_STRIP   ${CROSS_COMPILE}strip   PATHS ${TOOLCHAIN_HOME} NO_DEFAULT_PATH)
 
 find_program(CMAKE_GDB     ${CROSS_COMPILE}gdb     PATHS ${TOOLCHAIN_HOME} NO_DEFAULT_PATH)
+
+if (CMAKE_GDB)
```

Comment:
How do we handle the ARM embedded toolchain case in which the default gdb does NOT support python and they have `arm-none-eabi-gdb-py`

Reply:
For now, we handle it by just using that `arm-none-eabi-gdb`  since it doesn't fail to load. After 2.7.0 as I noted I plan to add `--gdb-py` and `--gdb-no-py` to `west debug` where the user will be able to select which one they want.